provider/rfc2136/rfc2136.go

```diff
@@ -221,14 +221,14 @@ func (r rfc2136Provider) ApplyChanges(ctx context.Context, changes *plan.Changes
 
 		r.AddRecord(m, ep)
 	}
-	for _, ep := range changes.UpdateNew {
+	for index, ep := range changes.UpdateNew {
```

Comment:
nit, since it only used it this small loop it should be i

Reply:
Fixed